src/index.ts

```diff
@@ -830,11 +831,25 @@ export type PreparedFn<T extends Dictionary<ParameterAlias>, U> = (
 	passthrough?: Params['passthrough'],
 ) => U;
 
+export type RetryParameters = {
```

Comment:
I think this should probably become
```suggestion
export type RetryParameters = false | {
```
where `false` is used to disable the retrying at an individual request level even if it's enabled at the module level. There may also be value in `true` to enable it with default options if we decide on some defaults that would be generally sensible